vault/provider.go

```diff
@@ -157,6 +158,25 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("VAULT_NAMESPACE", ""),
 				Description: "The namespace to use. Available only for Vault Enterprise",
 			},
+			"headers": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "The headers to send with each Vault request.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The header name",
+						},
+						"value": {
```

Comment:
Would it be possible to get test coverage of the  `name` and `value` fields? It would be cool to see what the Terraform config looks like when they're populated.

Reply:
I copied the basic token name test, and just added a custom provider configuration which included the headers.  It doesn't test any header related functionality (which would depend on individual use cases), only that the headers don't cause errors when Vault is doing things (in the case of this test, token lookup)